### DISCUSSION TITLE
VistaFolderBrowserDialog show files in folder

### DISCUSSION THREAD

**Discussion by MatzeFricke** [ookii-dialogs/ookii-dialogs-wpf#91] [Q&A]
Hi,

is there currently any way to show the content (the files inside) of a folder, while browsing for one in the VistaFolderBrowserDialog? Or to use any other of the File browsing dialogs to let a user select either file or folder?

For context: i'm currently showing a VistaFolderBrwoserDialog to import an image folder. But its not that easy to use if you're browsing an Nikon memory card, that has lots of folders and you can't see the content while picking one.

Thanks in advance (and many thanks for the great project)
Matze

**Comment by kabua** [comment]
One way would be to use the Load File Browser and then use only the path (i.e. discard the filename)

**Comment by niprst** [comment]
I tried this lib to have this exact functionality as well. Unfortunately it is the exact same behaviour of standard one.

The file browser forces to select a file in the folder which is counter-intuitive when you need to select the current folder (My goal is to select a folder to analyze its files)
And it is way more convenient to be in the folder and see its content rather than go back one level and selecting the folder.

The fusion of the two modes would be what is needed : being in a folder and being able to see files without selecting them.
